runtime/drivers/drivers.go

```diff
@@ -32,16 +33,20 @@ func Register(name string, driver Driver) {
 	Drivers[name] = driver
 }
 
+type OpenDataBucketFn func(ctx context.Context, instanceID string, elem ...string) (*blob.Bucket, error)
+
+var OpenNilDataBucket OpenDataBucketFn = func(ctx context.Context, instanceID string, elem ...string) (*blob.Bucket, error) { return nil, nil }
```

Comment:
It would be nice to avoid returning a `nil` value and `nil` error. The usual expectation is that after checking `if err != nil`, you are free to access a returned value. There are exceptions, but they often require extra caution.

Here, it can easily be fixed by either:
1. Returning `bucket, ok, err := ...`
2. Returning a constant `drivers.ErrBucketNotAvailable` that the caller can check for